### DISCUSSION TITLE
Native Influxdb support w/ Flux

### DISCUSSION THREAD

**Discussion by danmx** [ToolJet/ToolJet#2437] [Ideas]
It would nice to see Influxdb integration with support of Flux query.

**Comment by withshubh** [comment]
Hey @danmx 👋 
Thanks for creating this discussion and we would love to add InfluxDB integration. Can you please tell us a bit about your use case with this integration?

**Reply by danmx** [reply]
Hi @withshubh
I'm using Influxdb to store some analytics data. I'm looking for a low-code tool to create custom BI dashboards that can be extended in the future. Grafana works for a PoC but it can't really be personalised and customised to the level that I need.

**Reply by withshubh** [reply]
Makes sense. Created issue #2441

**Comment by Navaneeth-pk** [comment]
Just merged the integration, it will be included in the next release.
PR: https://github.com/ToolJet/ToolJet/pull/2590
